### DISCUSSION TITLE
A space insertion tool

### DISCUSSION THREAD

**Discussion by francoisschwarzentruber** [tableaunoir/tableaunoir#132] [Ideas]
You forgot to type a word?
You want to insert a panel?
You want to add an illustration?

It would be very nice to have an intuitive tool to insert space. Not clear yet how to do that.

**Comment by Naereen** [comment]
What do you mean "insert space" ?
Insert a vertical panel *between two other panels*?

**Comment by francoisschwarzentruber** [comment]
For instance, yes. Actually, when I teach, I never really experience this need. May this issue is not very important. Or maybe we should convert this issue in a discussion?

**Comment by tekhnee** [comment]
@francoisschwarzentruber I actually think this could be one of the most imaginative features. How to do this ergonomically and intuitively is a good question.

What if the user would define an ellipsis or rectangle (center point, width, height), whereby Tableaunoir would "push out" any objects that cross into that closed area? I am suggesting ellipses or rectangles because it only takes a single click-and-drag gesture to define them.
